core/src/main/java/io/aiven/klaw/listeners/MetadataListener.java

```diff
@@ -3,7 +3,7 @@
 import io.aiven.klaw.config.ManageDatabase;
 import io.aiven.klaw.model.KwMetadataUpdates;
 import io.aiven.klaw.model.enums.EntityType;
-import javax.persistence.*;
+import jakarta.persistence.*;
 import lombok.RequiredArgsConstructor;
```

Comment:
.* breakout to individual imports.

Reply:
Like mentioned earlier, there are more than 20 imports and hence this.